app/assets/javascripts/controllers/controllers.js

```diff
@@ -60,6 +60,16 @@ angular.module('barter')
     UserService.update(user);
   };
 
+  this.updateTalent = function(talent) {
+    console.log(talent);
+    $http.put('/talents/' + talent.id, talent)
+    .success(function(response){
+      console.log(response);
+      this.loadUserGraph();
```

Comment:
Be careful here - "this" is not bound to the controller unless you add .bind(this) to the end of the function body in line 69. So you'll probably get an error - js will evaluate loadUserGraph, see it's undefined, try to call it and report "undefined is not a function"